tower-http/CHANGELOG.md

```diff
@@ -33,24 +53,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   `Box<dyn std::error::Error + Send + Sync>`. This makes them usable if
   the body they're wrapping uses `Box<dyn std::error::Error + Send + Sync>` as
   its error type which they previously weren't ([#166]) (BREAKING)
```

Comment:
take it or leave it: it _might _be worth also having a separate section listing all of the breaking changes, as well? not a blocker.

Reply:
Hehe I actually got this format from tracing's changelog. Still haven't found one that I think is great but a separate section for breaking changes sounds good! I'll update it later.